Makefile

```diff
@@ -77,6 +77,10 @@ freeze-requirements: ## Pin all requirements including sub dependencies into req
 	pip install --upgrade pip-tools
 	pip-compile requirements.in
 
+.PHONY: bump-utils
+bump-utils:  # Bump notifications-utils package to latest version
+	${PYTHON_EXECUTABLE_PREFIX}python -c "from notifications_utils.version_tools import upgrade_version; upgrade_version()"
```

Comment:
`PYTHON_EXECUTABLE_PREFIX` isn't defined in this makefile yet, so will need to add it. In admin it comes from:

```
VIRTUALENV_ROOT := $(shell [ -z $$VIRTUAL_ENV ] && echo $$(pwd)/venv || echo $$VIRTUAL_ENV)
PYTHON_EXECUTABLE_PREFIX := $(shell test -d "$${VIRTUALENV_ROOT}" && echo "$${VIRTUALENV_ROOT}/bin/" || echo "")
```

Reply:
Noted and added in [8cad60a](https://github.com/alphagov/notifications-api/pull/3669/commits/8cad60a89964b75a345d03fe721a076af1579a03)